corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -46,7 +46,7 @@ def get_commcare_users_by_filters(domain, user_filters, count_only=False):
 
     if count_only:
         return query.count()
-    user_ids = [u['_id'] for u in query.source(['_id']).run().hits]
```

Comment:
Farid and I recently discovered that on our old version of ES, scroll queries don't preserve order.  If you want ordering here, you could use `query.get_ids()`.

Reply:
I don't think order is important here since there is no sort clause in the query.